src/Libraries/CoreNodeModels/Equals.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Dynamo.Graph.Nodes;
+using Newtonsoft.Json;
+using ProtoCore.AST.AssociativeAST;
+using ProtoCore.DSASM;
+using ProtoCore.Utils;
+
+namespace CoreNodeModels
+{
+    [NodeName("==")]
+    [NodeDescription("Equals with tolerance input")]
+    [NodeCategory("Core.Math")]
+    [NodeSearchTags("Equals")]
+    [InPortTypes("double ", "double", "double")]
+    [OutPortTypes("bool")]
+    [IsDesignScriptCompatible]
+    public class Equals : NodeModel
+    {
+        private static readonly string functionName = nameof(DSCore.List.Equals);
```

Comment:
did you want to use this in the AST?

Reply:
This is not used, I don't think. Ah, I see this is the wrong function. Will clean it up.